03 - Showcases & Templates/Vaults/Template_Hub.md

```diff
@@ -0,0 +1,49 @@
+---
+aliases: Template_Hub
+- 
+tags:
+- seedling
+- Template_Hub
```

Comment:
Please don't add new tags. We only use them as placeholders or to indicate the status of the note. See [Tag Glossary](https://github.com/obsidian-community/obsidian-hub/blob/main/00%20-%20Contribute%20to%20the%20Obsidian%20Hub/Tag%20glossary.md)

Reply:
done